src/tests/ftest/util/soak_utils.py

```diff
@@ -1199,9 +1200,20 @@ def create_app_cmdline(self, job_spec, pool, ppn, nodesperjob):
     """
     commands = []
     app_params = os.path.join(os.sep, "run", job_spec, "*")
-    app_cmd = os.path.expandvars(self.params.get("cmdline", app_params, default=None))
     mpi_module = self.params.get("module", app_params, self.mpi_module)
     api_list = self.params.get("api", app_params, default=["DFS"])
+    apps_dir = os.environ["DAOS_TEST_APP_DIR"]
+    # Update DAOS_TEST_APP_DIR if used in the cmdline param in yaml
+    # ${DAOS_TEST_APP_SRC}                  =>  apps built with el8 and mpi/mpich (default)
+    # ${DAOS_TEST_APP_SRC}/intelmpi         =>  apps built with el8 and intelmpi
```

Comment:
> GitHub Actions / Pylint check
>
> wrong-spelling-in-comment, Wrong spelling of a word 'intelmpi' in a comment:

Either remove this line are you can put this above it
```
# pylint: disable-next=wrong-spelling-in-comment,fixme
```

(And two more instances below)

Reply:
Sorry, I didn't mean to copy-past the `fixme` part :(
It should be
`# pylint: disable-next=wrong-spelling-in-comment`